app/components/press-release/sources-modal.hbs

```diff
@@ -0,0 +1,33 @@
+<Auk::Modal>
+  <Auk::Modal::Header
+    @title="Contactpersoon toevoegen"
+    @onClose={{@onCancel}}
+  />
+  <Auk::Modal::Body>
+    <div class="auk-form-group-layout auk-form-group-layout--standard">
+      <div class="auk-form-group">
+        <Auk::Label for="publication-date">
+          Contactpersonen
+        </Auk::Label>
+        <PowerSelect
```

Comment:
Een multi-select waarmee meerdere contacten in 1 keer toegevoegd kunnen worden is misschien gebruiksvriendelijker?

Reply:
Dat vond ik ook tijdens het coderen, maar in ticket stond een contact, ik zal dit aanpassen naar multiselect